### DISCUSSION TITLE
qpdf 11.0.0

### DISCUSSION THREAD

**Discussion by jberkenbilt** [qpdf/qpdf#762] [General]
This is qpdf version 11.0.0. This is a major release with lots of changes. For a full list of changes from previous releases, please see the [release notes](https://qpdf.readthedocs.io/en/stable/release-notes.html). See also [README-what-to-download](./README-what-to-download.md) for details about the available source and binary distributions. Here are some release highlights:
* qpdf JSON v2 -- a complete, unambiguous representation of PDF file contents using JSON with bidirectional conversion between PDF and JSON
* the build now uses cmake
* `PointerHolder` has been replaced by `std::shared_ptr`
* Many performance improvements
* Better memory safety for `QPDFObjectHandle` objects associated with destroyed `QPDF` objects
* A new logging facility that enables more reliable output/error redirection for library users

<hr /><em>This discussion was created from the release <a href='https://github.com/qpdf/qpdf/releases/tag/v11.0.0'>qpdf 11.0.0</a>.</em>

**Comment by atiq-cs** [comment]
Not sure where to put the thank you note.

"Just wanted to thanks so much for this pdf tool. I used to use pdftk on Windows, was having trouble on Linux to find a similar tool. This works awesome. So, Thank you!"

**Reply by jberkenbilt** [reply]
The thanks are appreciated regardless of where you place them. I'm glad you find it useful. :-)
